src/Shared/BrowserTesting/src/Microsoft.AspNetCore.BrowserTesting.csproj

```
@@ -6,6 +6,7 @@
     <IsShippingPackage>false</IsShippingPackage>
     <AddPublicApiAnalyzers>false</AddPublicApiAnalyzers>
     <IsTestAssetProject>true</IsTestAssetProject>
+    <DefineConstants Condition="'$(IsPlaywrightAvailable)' != 'true'">$(DefineConstants);DISABLE_PLAYWRIGHT</DefineConstants>
```

Doesn't this disable our tests by default? Where does `IsPlaywrightAvailable` come from?

Shouldn't this be
```suggestion
    <DefineConstants Condition="'$(IsPlaywrightAvailable)' == 'false'">$(DefineConstants);DISABLE_PLAYWRIGHT</DefineConstants>
```

---

It comes from the top-level `Directory.Build.props`:

https://github.com/dotnet/aspnetcore/blob/afd72717c019eb2b8f4e56ad39d1898a6d333fbc/Directory.Build.props#L180-L181

---

Though it was functional, I've committed your suggestion as it is safer if something would happen to the top-level `IsPlaywrightAvailable`.

---

Thanks!